### ISSUE TITLE
Broken link at https://docs.microsoft.com/en-us/windows/win32/controls/add-tree-view-items

### ISSUE THREAD

**Issue by guybark** [MicrosoftDocs/feedback#2376]
**🛑 IMPORTANT**: You can get your feedback addressed faster if you **use the comment section for the article in which you encountered a problem** (if available).

**Link to article:**
https://docs.microsoft.com/en-us/windows/win32/controls/add-tree-view-items

**Problem:**
Click the "CustDTv sample illustrates custom draw in a Tree-View control" link at the bottom of the article, and get unexpectedly taken to https://support.microsoft.com/en-us/

**Comment by HackerDaGreat57** [comment]
[Found it!](https://www.betaarchive.com/wiki/index.php/Microsoft_KB_Archive/248496)

**Comment by gopakumar-gg** [comment]
> [Found it!](https://www.betaarchive.com/wiki/index.php/Microsoft_KB_Archive/248496)

But, www.betaarchive.com is a third party site; not related to Microsoft.  Right?

**Comment by HackerDaGreat57** [comment]
> > [Found it!](https://www.betaarchive.com/wiki/index.php/Microsoft_KB_Archive/248496)
> 
> But, [www.betaarchive.com](http://www.betaarchive.com) is a third party site; not related to Microsoft. Right?

Correct, however it's better than nothing. I don't even remember what that page was for and I have long since moved on from Win32 programming, but I think the page looked different from what it looks like now when I originally linked to it.
